playground/docker-compose.fork.yml

```diff
@@ -127,6 +128,9 @@ services:
       - ./driver.toml:/driver.toml
     ports:
       - 9000:80
+    depends_on:
+      chain-check:
+        condition: service_healthy
```

Comment:
This condition is required since there is a time gap between the chain's container `created` status and when the anvil starts the service. This avoids adding a startup timeout to the driver container.